### PR TITLE
Combined compliance question mapping updates

### PR DESCRIPTION
**Pull request by jzolo22**

### CODE REVIEW

File: jupiterone/questions/questions.yaml [hunk 1]
```diff
@@ -173,10 +207,27 @@ questions:
...
+  - standard: CIS Google Cloud Platform Foundation Benchmark 2.0.0
+    version: v2.0.0
+    requirements:
+      - '1.4' 
+  - standard: CIS Google Cloud Foundations 1.3
+    requirements:
+      - '1.4'
+  - standard: CIS Google Cloud Foundations 2.0
     requirements:
       - '1.4'
```
Comment (jzolo22) [inline jupiterone/questions/questions.yaml:227]: This one seems worth another review. Are these duplicated at all?

File: jupiterone/questions/questions.yaml [hunk 2]
```diff
@@ -365,6 +474,146 @@ questions:
...
+    query: |
+      FIND google_dataproc_cluster WITH encrypted != true OR kmsKeyName !~= "cmek"
+  tags:
+  - google-cloud
+  - customer-managed-encryption-key
+  - encryption
+  compliance:
+  - standard: CIS Google Cloud Foundations 1.1
+    requirements:
+      - '1.17'
```
Comment (jzolo22) [inline jupiterone/questions/questions.yaml:610]: The questions spreadsheet doesn't have this question for 1.1, do we want to keep it or remove?

File: jupiterone/questions/questions.yaml [hunk 3]
```diff
@@ -365,6 +466,121 @@ questions:
...
+#     requirements:
+#       - '1.12'
+#   - standard: CIS Google Cloud Platform Foundation Benchmark 2.0.0
+#     requirements:
+#       - '1.12'
+
+
+#1.13 TBD
+#1.14 TBD
+#1.15 TBD
```
Comment (jzolo22) [inline jupiterone/questions/questions.yaml:531]: In this [ticket](https://jupiterone.atlassian.net/browse/APP-14216?atlOrigin=eyJpIjoiZWIxMzU5Y2MzMzMxNGZhNWFiNGY5Yjc0ZWE5NGU2M2YiLCJwIjoiaiJ9) it seems like the customer is expecting these ones, and it looks like we have them on the GCP spreadsheet. Any chance we can get them in?

**Review by mikiodehartj1 — APPROVED** [review]

File: jupiterone/questions/questions.yaml (hunk 3 above)
Comment (mikiodehartj1) [reply jupiterone/questions/questions.yaml:531]: These entities are currently not queryable/in the data model. Feature requests are open to bring these in

**Review by hallsamuel90 — APPROVED** [review]

**Pull request merged**